Cargo.toml

```diff
@@ -29,6 +30,7 @@ uuid = { version = "1.1", features = ["serde", "v4"] }
 reqwest = { version = "0.11.10", features = ["json"] }
 dotenv = { version = "0.9" }
 tempfile = { version = "3.3.0" }
+num-rational = { version = "0.4.1", features = ["serde"] }
```

Comment:
Do you need this? `squire_lib` should be re-exporting any necessary `r64`.

Reply:
You're right. I will import from `squire_lib`.